src/Sylius/Bundle/AdminBundle/Resources/views/Promotion/_toolbar.html.twig

```diff
@@ -0,0 +1,6 @@
+{% set menu = knp_menu_get('sylius.admin.coupon.manage', [], {'promotion': promotion}) %}
+<div class="ui right floated buttons">
+    {% if promotion.couponBased == true  %}
```

Comment:
Maybe we can move this `if` a level up? We for sure don't want to have an empty `<div class="ui right floated buttons"></div>` if the promotion is not coupon based ;)